parser-typechecker/src/Unison/CommandLine/InputPatterns.hs

```diff
@@ -1014,6 +1014,9 @@ replaceEdit f s = self
       _ -> Left $ I.help self
     )
 
+replace :: InputPattern
+replace = replaceEdit Input.ReplaceI "any"
```

Comment:
I really want this command to be called `replace`. In that case we could just delete the `replace.term` and `replace.type` commands, or implement them as a call to `replace` that errors if the names given aren't the appropriate sort.